src/content/docs/curriculum-file-structure.mdx

```diff
@@ -100,9 +175,9 @@ Also, you will likely want to rename the certificate and the `{superBlock}-proje
 
 <Steps>
 
-1. Rename the superBlock folder in the `curriculum/challenges/english` directory.
-2. Rename the superBlock folder in _all_ other `curriculum/challenges/{language}` directories.
-3. For each block within that superBlock, update the `superBlock` value in the `meta.json` file to its dashedName. You don't need to rename any folders here. Do that when renaming a block.
+1. Rename the superblock file in `curriculum/structure/superblocks/` from `{old-name}.json` to `{new-name}.json`.
```

Comment:
The `superBlockNames` map in curriculum/build-curriculum.js will need adjusting, too. If the structure filename changes, the map's key needs to change and if the enum value changes, the map's value needs to.